### PR TITLE
x11vnc: Update test check string

### PR DESCRIPTION
**Pull request by dj-m**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

Updated test block to use `system "#{bin}/x11vnc", "--version"

### CODE REVIEW

File: Formula/x11vnc.rb [hunk 1]
```diff
@@ -50,7 +50,7 @@ def install
 --- a/x11vnc/solid.c
 +++ b/x11vnc/solid.c
 @@ -177,7 +177,7 @@ unsigned long get_pixel(char *color) {
- 
+
```
Comment (DomT4) [inline Formula/x11vnc.rb]: Careful of accidentally tweaking patch whitespace. It's an excellent way to accidentally break things. (I've been trying to phase out larger inline patches for this very reason).

File: Formula/x11vnc.rb [hunk 2]
```diff
@@ -25,7 +25,7 @@ def install
       "--disable-debug",
       "--disable-dependency-tracking",
       "--prefix=#{prefix}",
-      "--mandir=#{man}"
+      "--mandir=#{man}",
```
Comment (DomT4) [inline Formula/x11vnc.rb]: If you wouldn't mind, the same comment I made IRT using a `%W` style array in the other PR would be great here as well.

**Comment by dj-m** [comment]
@DomT4 Weird, I thought I only touched the test block when I went mucking around in there... 🤔 Dat' install array, tho' 🙌🏾 🔥🎆

File: Formula/x11vnc.rb [hunk 3]
```diff
@@ -50,7 +50,6 @@ def install
 --- a/x11vnc/solid.c
 +++ b/x11vnc/solid.c
 @@ -177,7 +177,7 @@ unsigned long get_pixel(char *color) {
-
```
Comment (DomT4) [inline Formula/x11vnc.rb]: Heh, I think you've completely removed the whitespace here now. Looking at the original patch there's a single space on this line.

Sorry to be a pain about this 😢. Too much experience with minor whitespace changes in patches causing user breakage.

**Comment by dj-m** [comment]
@DomT4 No pain at all! Thanks for the info & learning opp... It was totally my oversight to not notice the whitespace in the actual path if/else statement.

**Pull request closed without merging**